app/models/grda_warehouse/tasks/chronically_homeless.rb

```diff
@@ -52,6 +52,48 @@ def initialize(
       @debug = debug
     end
 
+    def chronic_on_date(client_id)
+      debug_log "Calculating chronicity for #{client_id}"
+      # All chronically homeless must also be disabled
+      return false unless disabled?(client_id)
```

Comment:
What if the client was disabled on `@date`? We only check if currently disabled. I think maybe we should be passing in a date to the `chronically_disabled` scope on line 352 of this file.

Reply:
That's probably true.  The spec for this hasn't evolved with the rest of the warehouse.  Historically, this has been more lenient in determining disability status.